src/scripts/main.js

```diff
@@ -1 +1,60 @@
+/* eslint-disable prefer-promise-reject-errors */
 'use strict';
+
+function createMessage(message, className) {
+  document.body.insertAdjacentHTML(
+    'afterbegin',
+    `
+    <div data-qa="notification" class="${className}">
+    ${message}
+    </div>
+    `,
+  );
+}
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve();
+  });
+
+  setTimeout(() => {
+    reject();
```

Comment:
reject(new Error(''));

/* eslint-disable prefer-promise-reject-errors */ => this rule should not be disabled